eng/make-bootstrap.ps1

```diff
@@ -62,6 +62,10 @@ try {
 
   if ($ci) {
     $args += " /p:ContinuousIntegrationBuild=true"
+  
+    # Since this build is running in CI we need to set the nuget package directory so that we restore locally.
+    $env:NUGET_PACKAGES = Join-Path $RepoRoot '.packages\'
```

Comment:
Arcade SDK sets the NuGetPackageRoot MSBuild property for CI builds to this path. However, the Restore task runs and does not take the MSBuild property into account. Restore does respect the NuGet environment variables however.